dist/js/select2.full.js

```diff
@@ -2019,7 +2027,13 @@ S2.define('select2/selection/search',[
 
     this.resizeSearch();
     if (searchHadFocus) {
-      this.$search.focus();
+      var isTagInput = this.$element.find('[data-select2-tag]').length;
+      if (isTagInput) {
```

Comment:
To make this work in IE11 with multiple tag support, I had to combine this code with the code presented in a comment on https://github.com/select2/select2/issues/4398:
```var isTagInput = this.$element.find('[data-select2-tag]').length;
if (isTagInput) {
    // fix IE11 bug where tag input lost focus
    window.setTimeout(function() {
        self.$element.focus();
    }, 0);
} else {
    window.setTimeout(function() {
        self.$search.focus();
    }, 0);
}

Reply:
This one worked for me